verifiers/internal/gha/slsaprovenance/v1.0/container_based.go

```diff
@@ -1,11 +1,188 @@
 package v1
 
-// containerBasedBuildType is the build type for the container-based builder and is based on BYOB.
-var containerBasedBuildType = "https://slsa.dev/container-based-build/v0.1?draft"
+import (
+	"fmt"
+	"strings"
+	"time"
 
-// ContainerBasedProvenance is provenance generated by the container-based builder.
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
+	slsa1 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
+	serrors "github.com/slsa-framework/slsa-verifier/v2/errors"
+
+	"github.com/slsa-framework/slsa-verifier/v2/verifiers/internal/gha/slsaprovenance/common"
+)
+
+// ContainerBasedProvenance is SLSA v1.0 provenance for the slsa-github-generator BYOB build type.
 type ContainerBasedProvenance struct {
```

Comment:
Note to reviewers: ContainerBasedProvenance doesn't support sha1 and only supports source == trigger